package.yaml

```diff
@@ -1,5 +1,5 @@
 name: simformat
-version: 0.0.0.1
+version: 0.0.0.2
```

Comment:
This is an upgrade to the (command-line) interface, so strictly speaking I guess it should be a minor version bump?